backend/controllers/userController.js

```diff
@@ -0,0 +1,30 @@
+const { QueryTypes } = require('sequelize');
+const db = require('../config/database');
+
+const getAllRoles = async (req, res, next) => {
+  const roles = await db.query(
+    'SELECT AccountId, FirstName, LastName, Role FROM MinicapDatabase.dbo.Users t',
```

Comment:
we should agree on using either only sequelize, or only writing the queries ourselves, having a mix of both makes things a bit messy